<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>newWindow</class>
 <widget class="QWidget" name="newWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>300</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <bold>true</bold>
   </font>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>数据处理设置</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="3">
    <widget class="QPushButton" name="buttonOutputData">
     <property name="text">
      <string>导出原始数据</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../svgsource/logout-circle-r-line.svg</normaloff>../../svgsource/logout-circle-r-line.svg</iconset>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="pushButton_2">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>滤波等级</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QCheckBox" name="boxWaveShapeCorrect">
     <property name="text">
      <string>波形矫正</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="3">
    <widget class="QSpinBox" name="spinSmoothLevel">
     <property name="maximum">
      <number>4</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>数据处理</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../svgsource/line-chart-line.svg</normaloff>../../svgsource/line-chart-line.svg</iconset>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QCheckBox" name="boxNoLinearCorrect">
     <property name="text">
      <string>非线性矫正</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QCheckBox" name="boxDeductDark">
     <property name="text">
      <string>暗底矫正</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <property name="tristate">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QPushButton" name="buttonCorrect">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../svgsource/check-line.svg</normaloff>../../svgsource/check-line.svg</iconset>
     </property>
    </widget>
   </item>
   <item row="5" column="2" colspan="2">
    <widget class="QPushButton" name="buttonCancel">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../svgsource/close-line.svg</normaloff>../../svgsource/close-line.svg</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
